catalog/github/README.md

```
@@ -0,0 +1,142 @@
+| `url` | Url where the label was used | `issue_labels.url` | 🟢 _1.00_ | *Direct URL mapping, both refer to the location of usage.* |
+
+### Mapping: Airbyte `users` to Fivetran `user`
+
+
+- Table Match Confidence Score: 🟢 _0.80_
+- Table Completion Score: ⚠️ _0.50_
+- Summary Self-Evaluation: _The table match score is high indicating a good correlation between source and target tables. However, the completion score is lower due to missing field mappings._
+
+| Field | Description | Expression | Confidence | Evaluation |
+| --- | --- | --- | --- | --- |
+| `id` | System generated unique id for the user. | `users.id` | 🟢 _1.00_ | *Direct match found: 'users.id' matches 'id'.* |
```

markdownlint-cli2 (0.17.2)

Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)